.coveragerc

```diff
@@ -12,10 +13,13 @@ exclude_lines =
     raise AssertionError
     raise NotImplementedError
     raise TypeError
+    raise InterpolationAmbiguityError
+    raise InterpolationAttemptingFullContextError
+    raise InterpolationContextSensitivityError
```

Comment:
Ok -- that's weird though, looks like you are looking at an old version of the diff, make sure you are on the latest version I pushed!

Reply:
(Almost) done in fea5169: since I had a base `GrammarParseError` inheriting from `OmegaConfBaseException` I thought it would make sense to inherit from it instead -- but I'm not 100% sure if your goal was to get rid of mostly unused error classes or if you really wanted them explicitly not to be considered as ``GrammarParseError`s ==> please just confirm whether that's ok